### DISCUSSION TITLE
Does ldc2 and ldc need to have a same version ?

### DISCUSSION THREAD

**Discussion by hsadeghi** [ldc-developers/ldc#4512] [Q&A]
Hi all,

I have built ldc1.32 with ldc2 v1.26 (and llvm 15.0.7) and it sometimes generates wrong binaries (target arch: RISCV64), so I was wondering if they need to a same version ? (I also have ldc1.30 built with ldc2 v1.26 and llvm 14.0 which works fine btw).

Many thanks for your reply,

Hadi

**Comment by kinke** [comment]
Nope, the host compiler version does not need to match the compiler being built. ~Any non-ancient version should work.

**Reply by hsadeghi** [reply]
yes, I tried ldc_1.32  with both  ldc2_1.26 and ldc2_1.32 and the issue remains

**Reply by JohanEngelen** [reply]
can you be more exact in your LDC version descriptions? (full LDC version and LLVM version, basically the first few lines of output from `ldc --version`)
